lance-lakekeeper-test/test_lance_lakekeeper.py

```
@@ -0,0 +1,577 @@
+
+@pytest.fixture()
+def ns(catalog, request):
+    """Create a namespace and drop it after the test."""
+    name = f"test_{request.node.name}_{uuid.uuid4().hex[:8]}"
+    catalog.create_namespace(name)
+    yield name
+    for t in catalog.list_tables(name):
+        try:
+            catalog.drop_table(t)
+        except Exception:
+            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)